src/scripts.js

```diff
@@ -18,16 +19,34 @@ const ingredientsList = document.querySelector('.ingredients-list');
 const directionsList = document.querySelector('.directions-list');
```

Comment:
Let's decide on a convention to querySelect/getElementById to apply consistency.